examples/resources/alkira_peering_gateway_cxp/resource.tf

```diff
@@ -0,0 +1,7 @@
+resource "alkira_peering_gateway_cxp" "test1" {
+  name         = "tf-test-1"
```

Comment:
For example, we try to move away from referring to our internal test naming. I am fixing all other examples. so we could just use name `example`. Tweak all `test` to `example`.

Reply:
done.